spec/grid_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "spec_helper"
+require_relative "../lib/grid.rb"
+
+RSpec.describe Grid do
+  describe "#coordinate_valid?" do
```

Comment:
I think you broke out a lot of error cases, and some of the success cases, but in addition to the last row and column, what else defines the boundaries of the grid?

Reply:
Hmm, I'll need to think about this..